homeassistant/components/sun/const.py

```diff
@@ -2,8 +2,12 @@
 
 from typing import Final
 
+from homeassistant.const import Platform
+
 DOMAIN: Final = "sun"
 
+PLATFORMS = [Platform.BINARY_SENSOR, Platform.SENSOR]
```

Comment:
Please define it in the package module since it's only used in that module. Only shared constants should be defined in const.py.